### PR TITLE
Mamba poetry

### PR DESCRIPTION
**Pull request by dwu359**
[https://github.com/DSGT-DLP/Deep-Learning-Playground/pull/963](https://github.com/DSGT-DLP/Deep-Learning-Playground/pull/963)

### CODE REVIEW

File: pkg/exec_bash_cmd.go [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
 	"syscall"
 
 	"github.com/creack/pty"
+	"github.com/pterm/pterm"
 	"golang.org/x/term"
 )
 
-func ExecBashCmd(dir string, name string, arg ...string) {
+func ExecBashCmd(runtime_os string, dir string, name string, arg ...string) string {
+	if runtime_os == "windows" {
```
Comment (karkir0003) [inline pkg/exec_bash_cmd.go:19]: confused on how I specify OS in dlp-cli command @dwu359? can you add that to the readme docs

File: cmd/backend/backend.go [hunk 2]
```diff
@@ -20,4 +20,9 @@ var BackendCmd = &cobra.Command{
 
 func init() {
 	cmd.RootCmd.AddCommand(BackendCmd)
+	BackendCmd.PersistentFlags().String("env-name", "dlp", "Name of the conda environment you want to create")
```
Comment (karkir0003) [inline cmd/backend/backend.go:23]: can we force the env name to be `dlp` for consistency? If so, maybe make the env name a constant?

**Review by karkir0003 — APPROVED** [review]
looks good to me @dwu359

File: pkg/exec_bash_cmd.go (hunk 1 above)
Comment (dwu359) [reply pkg/exec_bash_cmd.go:19]: -w

**Pull request merged**